citi-onboarding/Delta-odonto
# Hotfix/values and background

##What I did
-Fixed all the backgrounds
-Changed ine of the values

##How to test
Follow the steps
1- Access this branch on your computer and type on the terminal theses commands:
`cd client`
`yarn start`

2- Then open another terminal at the same time and type in there theses commands:
`cd server`
`node index.js`

3- After this wait a few seconds and type on your browser: [localhost:3001/admin](url)

4- The values and the background should appear as the mockup that can the found here [https://www.figma.com/file/WO1wTZDVcs145XzB3HRY5M/Odonto-OnePage?node-id=0%3A1](url)
